tests/unit/features/RandomFourierDotFeatures_unittest.cc

```diff
@@ -176,3 +176,30 @@ TEST(RandomFourierDotFeatures, add_to_dense_test)
 
 }
 
+TEST(RandomFourierDotFeatures, random_coeff_dim)
+{
+	int32_t num_dims = 50;
+	int32_t vecs = 5;
+	int32_t D = 100;
+
+	SGMatrix<int32_t> data(num_dims, vecs);
```

Comment:
could use a matrix full of constants to reduce code here if youre not checking for the results

Reply:
Hmm, how about we create a test_fixture? I mean there's a lot of repetition here.